src/styles/index.scss

```diff
@@ -1,3 +1,78 @@
-body {
-  margin: 0;
+@use 'sass:list';
+@import 'variables';
+
+.page {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  &__body {
+    margin: 0;
+    height: 100vh;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 10px;
+
+  // Final column width including the total width of days and the gaps between them
+  width: calc((#{$day-size} * #{$columns}) + (#{$gap} * #{$columns - 1}));
```

Comment:
The calculation for the width of the `.calendar` class seems to use a variable `$gap`, but it is not defined in the provided code. Ensure that `$gap` is defined in the `variables` file or elsewhere in the code.